pipeline_steps/holland.groovy

```diff
@@ -10,10 +10,18 @@ def holland(){
       / * recent versions of openstack-ansible allow execution of playbooks
         * from any directory, but that doesn't work all the way back to liberty
         */
-      sh """cp rpc-gating/playbooks/test_holland.yml /opt/rpc-openstack/rpcd/playbooks"""
+      
+      if (vm == null) {
+        sh "cp rpc-gating/playbooks/test_holland.yml /opt/rpc-openstack/rpcd/playbooks"
+      } else {
+        sh "scp -o StrictHostKeyChecking=no -p rpc-gating/playbooks/test_holland.yml deploy1:/opt/rpc-openstack/rpcd/playbooks"
+      }
+
       common.openstack_ansible(
+        vm: vm,
         playbook: "test_holland.yml",
-        path: "/opt/rpc-openstack/rpcd/playbooks"
+        path: "/opt/rpc-openstack/rpcd/playbooks",
+        enviornment_vars: ["ANSIBLE_CACHE_PLUGIN=memory"]
```

Comment:
Typo on `environment_vars`.  Since this isn't getting passed in correctly, is it actually needed?